### DISCUSSION TITLE
Membuat Tampilan Waktu Shalat

### DISCUSSION THREAD

**Discussion by nafiesl** [buku-masjid/buku-masjid#85] [General]
## Deskripsi

Ada user yang menyarankan untuk ditampilkan waktu shalat di halaman publik buku masjid. Contohnya seperti di website ini:
https://masjidassalaampmi.com

**Tampilan Desktop**

![screen_2024-08-14_001](https://github.com/user-attachments/assets/0c61659d-b53f-44d0-b9ee-9c787bb162ea)

**Tampilan Mobile**

![screen_2024-08-14_002](https://github.com/user-attachments/assets/3fb66cd4-adf1-4751-846e-da83ae1ee25d)

## Kemungkinan solusi

1. Buat template waktu shalat dengan bootstrap grid atau tabel
2. Sumber rujukan waktu shalat: Jadwal Shalat Kemenag: https://bimasislam.kemenag.go.id/jadwalshalat
3. Perlu pengaturan waktu shalat di halaman pengaturan admin
    - Pilih Provinsi dan Kota
    - Download jadwal shalat 2 bulan ke depan
    - Warning Admin jika jadwal shalat perlu di-update (1 pekan sebelum jadwal shalat habis)
4. Ada pengaturan on-off waktu shalat.

**Comment by 3xploiton3** [comment]
tidak begitu prioritas, 
terlalu ramai klo ditampilkan di buku masjid,
masih ok klo ditampilkan di homepage website masjid

**Reply by nafiesl** [reply]
Halo mas @3xploiton3, terima kasih masukannya. 

Fitur ini beberapa user yang mengusulkan sih mas, kebetulan saya juga tertarik untuk developmentnya. Saat ini kita belum ada fitur lain yang sifatnya lebih prioritas.
